main.py

```diff
@@ -223,43 +231,55 @@ async def votekick(update: Update, context: CallbackContext):
 
 	if tuser.id == 777000:
 		if (db.get_trusted(voter.id) or await is_admin(chat, voter)):
-			await update.message.reply_text(
+			reply = await update.message.reply_text(
 				"You can't votekick the channel…",
 				parse_mode=ParseMode.MARKDOWN_V2
 			)
+			db.add_vk_messages(tuser.id, [update.message.message_id, reply.message_id])

```

Comment:
no votekick going on, this would be to-delete immediately. we could have a timeout for these, but that'd be a separate PR and not hi-prio.

same with all the other "you can't votekick this or that" messages